build

```diff
@@ -58,10 +58,84 @@ create_aliases() {
   done <"$ALIASES"
 }
 
+create_hyprcursors() {
+  local src_dir="$1"
+  local out_dir="$2"
+  local theme_name="$3"
+  local hypr_dir="$out_dir/hyprcursors"
+  local filename dir cursor_dir striped base_name symlink target cursor_dir theme_comment
+
+  [ -d "$src_dir" ] || return 1
+  [ -d "$out_dir" ] || mkdir -p "$out_dir"
+  [ -d "$hypr_dir" ] || mkdir -p "$hypr_dir"
+
+  for file in "$src_dir"/*.svg; do
+    filename=$(basename -- "$file")
+
+    if [[ "$filename" == *_24.svg ]]; then
+      continue
+    fi
```

Comment:
Can you help me understand the reasoning for this `continue`? It's not obvious to me

Reply:
Since hyprcursor is SVG based, we don't need a special case for size 24 and 48, like is done for the xcursor portion. The hyprcursor documentation even says that the "size" parameter is ignored if using a SVG, so I just ignore the special case as it isn't really necessary. I could do the opposite and use the `_24` variant of each cursor as well and ignore the other. Only one of them is really necessary.

Reply:
Would using only the `_24` variant reduce the amount of iterations? We should only iterate across each SVG once if the size variants don't matter.